### ISSUE TITLE
Feature Request: Open original

### ISSUE THREAD

**Issue by wc8** [friebetill/obsidian-file-diff#2]
Thanks for this!

It would be handy if eventually the plugin could look for "sync-conflict" and open the related file name, i.e. the original to start a compare & merge.

**Comment by ryanwwest** [comment]
I second this, assuming I understand it correctly. At a wider level for Syncthing, it would be nice for it to autodetect conflicts and open up the diff resolver so you don't have to go figure out the filenames of conflicts.

**Comment by wc8** [comment]
Yes, that's the idea.

**Comment by friebetill** [comment]
I have extended the plugin by the command `Find sync conflicts and merge`.  The commando finds all Syncthing conflicts and shows the merge option for each conflict sequentially.

**Comment by wc8** [comment]
Thanks. 

It would help to have an indicator of the page being worked on, as in some cases the correction to be accepted depends on the file. (Maybe the info is there and I'm missing it?).

BTW, I'm no longer seeing the original commands in the command pallet only the new one, but maybe this was intentional and I'm misunderstanding the meaning of "extended".

**Comment by friebetill** [comment]
I showed the information which files are being differenced in the previous version, but had to deactivate it for now because I needed to make architecture changes based on the feedback of @liamcain. I am in dialogue with Liam to find a way to reactivate it.

Weird! I just downloaded the latest version again and I see all three options and unfortunately cannot reproduce it. The code also looks fine. Is the bug still present?
